src/main/webapp/app/exercises/shared/result/result.component.ts

```diff
@@ -518,4 +522,21 @@ export class ResultComponent implements OnInit, OnChanges {
     isManualResult(result: Result | undefined) {
         return result?.assessmentType !== AssessmentType.AUTOMATIC;
     }
+
+    /**
+     * Determines if some information about testcases could still be hidden because of later individual due dates
+     * @param componentInstance the detailed result view
+     */
+    private determineShowMissingAutomaticFeedbackInformation(componentInstance: ResultDetailComponent) {
+        if (!this.latestIndividualDueDate) {
+            this.exerciseService.getLatestDueDate(this.exercise!.id!).subscribe((latestIndividualDueDate?: dayjs.Dayjs) => {
+                this.latestIndividualDueDate = latestIndividualDueDate;
+                componentInstance.showMissingAutomaticFeedbackInformation = dayjs().isBefore(this.latestIndividualDueDate);
+                componentInstance.latestIndividualDueDate = this.latestIndividualDueDate;
+            });
+        } else {
+            componentInstance.showMissingAutomaticFeedbackInformation = dayjs().isBefore(this.latestIndividualDueDate);
+            componentInstance.latestIndividualDueDate = this.latestIndividualDueDate;
```

Comment:
Even if it's only two lines, we should move this into a method and not duplicate it.